tests/naming/src/lib.rs

```diff
@@ -63,3 +63,14 @@ fn f(x: Foobar) -> usize {
 fn reserved_names(val: u8, noeq: u8, of: u8) -> u8 {
     val + noeq + of
 }
+
+struct Arity1<T>;
```

Comment:
This is not a valid struct. `T` must be used somehow (at least with phantom data).

Reply:
Oh right! Thanks, was only compiling though Hax (I guess this check happens at MIR or something)